modules/rosetta/src/main/scala/org/tessellation/rosetta/server/model/search/SearchTransactionTransactionRequestOperator.scala

```diff
@@ -0,0 +1,32 @@
+package org.tessellation.rosetta.server.model.search
+
+import scala.util.Try
+
+import derevo.cats.{eqv, show}
+import derevo.derive
+import enumeratum.{Enum, EnumEntry}
+import io.circe.{Decoder, Encoder}
+
+/** This enumeration is for internal use only and meant to mirror:
+  *
+  * https://www.rosetta-api.org/docs/models/Operator.html
+  */
+
+@derive(eqv, show)
+sealed trait SearchTransactionTransactionRequestOperator extends EnumEntry
```

Comment:
imo it should be just `Operator` to mimic the rosetta specification. you already did that for other case classes that were auto-generated as empty and you created an enum for that. then you can reuse it in each search request. otherwise, you'd have to duplicate this code (because by its name it can be used for search transaction request only)

Reply:
Changed to `Operator`.